uktrade/data-hub-frontend
# Fix/imports

## Description of change

This PR extends the ESLint configuration from the [`import/recommended`](https://www.npmjs.com/package/eslint-plugin-import) plugin. It also fixes all the import issues that were picked up by introducing this plugin.

This should prevent bugs caused by wrong import like the #7147.

## Test instructions

1. In any file under `/src/client`
2. Try to import a non-existent name from an existing module, e.g `import {nonExistentName} from './existent-module'`
3. The `nonExistentName` should be be picked up as an `nonExistentName not found in ./existent-module` error by ESLint
